app/src/main/res/values/strings.xml

```diff
@@ -337,11 +337,19 @@
     <string name="receive_pay_and_countinue">Pay %s SOL &amp; Continue</string>
     <string name="receive_on_solana">%s on Solana</string>
     <string name="receive_my_sol_address">My Solana address</string>
+    <string name="receive_my_eth_address">My Ethereum address</string>
     <string name="receive_my_username">My username</string>
     <string name="receive_copy_address">Copy address</string>
     <string name="receive_sol_address_copied">✅ Your Solana address was copied</string>
     <string name="receive_eth_address_copied">✅ Your Ethereum address was copied</string>
     <string name="receive_username_copied">✅ Your username was copied</string>
+    <string name="receive_on_ethereum">%s on Ethereum</string>
```

Comment:
you can try and extract receive strings to separate strings.xml like we did in swap